src/streaming/protection/controllers/ProtectionController.js

```diff
@@ -417,6 +419,12 @@ function ProtectionController(config) {
                     for (let i = 0; i < pendingNeedKeyData.length; i++) {
                         for (ksIdx = 0; ksIdx < pendingNeedKeyData[i].length; ksIdx++) {
                             if (keySystem === pendingNeedKeyData[i][ksIdx].ks) {
+                                if (pendingNeedKeyData[i][ksIdx].initData === null && protData.hasOwnProperty('clearkeys')) {
+                                    var initData = { kids: Object.keys(protData.clearkeys) };
```

Comment:
Small fix. To avoid possible future issues related with variables scope, could you please declare initData using const instead of var?